<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>549</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPlainTextEdit" name="log">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QListWidget" name="todo_list">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QDateEdit" name="date_filter">
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="locale">
                 <locale language="Spanish" country="Spain"/>
                </property>
                <property name="displayFormat">
                 <string>dd/MM/yyyy</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="button_filter_today">
                <property name="text">
                 <string>t</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="filter">
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLineEdit" name="subject"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QSpinBox" name="priority">
                    <property name="minimum">
                     <number>0</number>
                    </property>
                    <property name="maximum">
                     <number>10</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTimeEdit" name="time">
                    <property name="focusPolicy">
                     <enum>Qt::WheelFocus</enum>
                    </property>
                    <property name="displayFormat">
                     <string>hh:mm</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="tags">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="tabChangesFocus">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QFontComboBox" name="fontComboBox">
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="button_bold">
                    <property name="font">
                     <font>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string>B</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="button_italic">
                    <property name="font">
                     <font>
                      <italic>true</italic>
                     </font>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string>I</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="button_font">
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string>Font</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="button_color">
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string>color</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="button_bgcolor">
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="text">
                     <string>bg color</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCalendarWidget" name="date">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="firstDayOfWeek">
                 <enum>Qt::Monday</enum>
                </property>
                <property name="gridVisible">
                 <bool>false</bool>
                </property>
                <property name="horizontalHeaderFormat">
                 <enum>QCalendarWidget::NoHorizontalHeader</enum>
                </property>
                <property name="verticalHeaderFormat">
                 <enum>QCalendarWidget::NoVerticalHeader</enum>
                </property>
                <property name="navigationBarVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextEdit" name="text">
              <property name="autoFormatting">
               <set>QTextEdit::AutoAll</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QPlainTextEdit" name="test">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>781</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_todo">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew_todo"/>
    <addaction name="actionDelete_todo"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionOrder"/>
   </widget>
   <widget class="QMenu" name="menuNavigate">
    <property name="title">
     <string>&amp;navigate</string>
    </property>
    <addaction name="actionUp"/>
    <addaction name="actionDown"/>
    <addaction name="separator"/>
    <addaction name="actionGoto_subject"/>
    <addaction name="actionGoto_filter"/>
   </widget>
   <widget class="QMenu" name="menuModif">
    <property name="title">
     <string>&amp;modif</string>
    </property>
    <addaction name="actionUp_priority"/>
    <addaction name="actionDown_priority"/>
    <addaction name="separator"/>
    <addaction name="actionInsert_date"/>
   </widget>
   <addaction name="menuNew_todo"/>
   <addaction name="menuNavigate"/>
   <addaction name="menuModif"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_todo">
   <property name="text">
    <string>&amp;new todo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete_todo">
   <property name="text">
    <string>&amp;delete todo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+D</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;reload</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="text">
    <string>&amp;up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="text">
    <string>&amp;down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionOrder">
   <property name="text">
    <string>&amp;order</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionGoto_subject">
   <property name="text">
    <string>&amp;goto subject</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Tab</string>
   </property>
  </action>
  <action name="actionUp_priority">
   <property name="text">
    <string>&amp;up priority</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDown_priority">
   <property name="text">
    <string>&amp;down priority</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionGoto_filter">
   <property name="text">
    <string>goto &amp;filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionInsert_date">
   <property name="text">
    <string>&amp;insert date</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>subject</tabstop>
  <tabstop>priority</tabstop>
  <tabstop>time</tabstop>
  <tabstop>date</tabstop>
  <tabstop>tags</tabstop>
  <tabstop>text</tabstop>
  <tabstop>button_filter_today</tabstop>
  <tabstop>filter</tabstop>
  <tabstop>test</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>priority_udpated(int)</slot>
 </slots>
</ui>
